This file and its contents are supplied under the terms of the
Common Development and Distribution License ("CDDL"), version 1.0.
You may only use this file in accordance with the terms of version
1.0 of the CDDL.

A full copy of the text of the CDDL should have accompanied this
source.  A copy of the CDDL is also available via the Internet at
http://www.illumos.org/license/CDDL.

Copyright (c) 2018, Joyent, Inc.

---

Without this patch, debug builds are unhappy about unused functions.

  cc1: warnings being treated as errors
  ../../intel/sys/acpi/acpixf.h:1203: error: 'AcpiDebugPrint' defined but not used [-Wunused-function]
  ../../intel/sys/acpi/acpixf.h:1215: error: 'AcpiDebugPrintRaw' defined but not used [-Wunused-function]
  ../../intel/sys/acpi/acpixf.h:1227: error: 'AcpiTracePoint' defined but not used [-Wunused-function]
  ../../intel/sys/acpi/acpixf.h:1235: error: 'AcpiLogError' defined but not used [-Wunused-function]
  *** Error code 1
  make: Warning: Command failed for target `debug64/psm_common.o'
  Current working directory
  .../usr/src/uts/i86pc/apix


diff --git b/usr/src/uts/intel/sys/acpi/acpixf.h a/usr/src/uts/intel/sys/acpi/acpixf.h
index aafac14..81e550a 100644
--- b/usr/src/uts/intel/sys/acpi/acpixf.h
+++ a/usr/src/uts/intel/sys/acpi/acpixf.h
@@ -363,9 +361,7 @@ ACPI_GLOBAL (BOOLEAN,               AcpiGbl_SystemAwakeAndRunning);
     Prototype;
 
 #else
-#define ACPI_DBG_DEPENDENT_RETURN_VOID(Prototype) \
-    static ACPI_INLINE Prototype {return;}
-
+#define ACPI_DBG_DEPENDENT_RETURN_VOID(Prototype)
 #endif /* ACPI_DEBUG_OUTPUT */
 
 
@@ -381,9 +377,7 @@ ACPI_GLOBAL (BOOLEAN,               AcpiGbl_SystemAwakeAndRunning);
     Prototype;
 
 #else
-#define ACPI_APP_DEPENDENT_RETURN_VOID(Prototype) \
-    static ACPI_INLINE Prototype {return;}
-
+#define ACPI_APP_DEPENDENT_RETURN_VOID(Prototype)
 #endif /* ACPI_APPLICATION */
 
 
